src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
Redirecting is indeed a good idea, the flash message annoys me as it would disallow caching on these pages.